src/main/resources/pdiresources/form/log-for-form-references.ktr

```diff
@@ -0,0 +1,1709 @@
+      <to>Merge join</to>
+      <enabled>Y</enabled>
+    </hop>
+    <hop>
+      <from>sort harmonized</from>
+      <to>Merge join</to>
+      <enabled>Y</enabled>
+    </hop>
+    <hop>
+      <from>Merge join</from>
+      <to>Select values</to>
+      <enabled>Y</enabled>
+    </hop>
+    <hop>
+      <from>Select values</from>
+      <to>Filter rows</to>
+      <enabled>Y</enabled>
+    </hop>
+    <hop>
+      <from>Filter rows</from>
+      <to>Dummy (do nothing)</to>
+      <enabled>Y</enabled>
+    </hop>
+    <hop>
+      <from>Dummy (do nothing)</from>
+      <to>Join rows (cartesian product)</to>
+      <enabled>Y</enabled>
+    </hop>
+    <hop>
+      <from>Dummy (do nothing)</from>
+      <to>Join rows (cartesian product) 2</to>
+      <enabled>Y</enabled>
+    </hop>
+    <hop>
+      <from>Dummy (do nothing)</from>
+      <to>Join rows (cartesian product) 3</to>
+      <enabled>Y</enabled>
+    </hop>
+    <hop>
+      <from>Dummy (do nothing)</from>
+      <to>Join rows (cartesian product) 4</to>
+      <enabled>Y</enabled>
+    </hop>
+    <hop>
+      <from>updated encounters</from>
+      <to>Join rows (cartesian product)</to>
+      <enabled>Y</enabled>
+    </hop>
+    <hop>
+      <from>update-htmlform</from>
+      <to>Join rows (cartesian product) 3</to>
+      <enabled>Y</enabled>
+    </hop>
+    <hop>
+      <from>updated form_resource</from>
+      <to>Join rows (cartesian product) 4</to>
+      <enabled>Y</enabled>
+    </hop>
+    <hop>
+      <from>updated form_field</from>
+      <to>Join rows (cartesian product) 2</to>
+      <enabled>Y</enabled>
+    </hop>
+    <hop>
+      <from>Join rows (cartesian product) 3</from>
+      <to>htmlform-flog</to>
+      <enabled>Y</enabled>
+    </hop>
+    <hop>
+      <from>Join rows (cartesian product)</from>
+      <to>encounter-log</to>
+      <enabled>Y</enabled>
+    </hop>
+    <hop>
+      <from>Join rows (cartesian product) 2</from>
+      <to>form_field log</to>
+      <enabled>Y</enabled>
+    </hop>
+    <hop>
+      <from>Join rows (cartesian product) 4</from>
+      <to>form_resource-log</to>
+      <enabled>Y</enabled>
+    </hop>
+  </order>
+  <step>
+    <name>get old form</name>
+    <type>TableInput</type>
+    <description/>
+    <distribute>Y</distribute>
+    <custom_distribution/>
+    <copies>1</copies>
+    <partitioning>
+      <method>none</method>
+      <schema_name/>
+    </partitioning>
+    <connection>OpenMRS source DB</connection>
+    <sql>SELECT
+  form_id old_form_id
+, name old_name
+, version old_version
+, build old_build
+, published old_published
+, description old_description
+, encounter_type old_encounter_type
+, template old_template
+, xslt old_xslt
+, creator old_creator
+, date_created old_date_created
+, changed_by old_changed_by
+, date_changed old_date_changed
+, retired old_retired
+, retired_by old_retired_by
+, date_retired old_date_retired
+, retired_reason old_retired_reason
+, uuid old_uuid
+, updated old_updated
```

Comment:
`updated` column is not added to metadata table, this causes the transformation to fail. Similar for other metadata.

Reply:
done